src/web/app/components/question-edit-form/question-edit-form.component.html

```diff
@@ -1,20 +1,24 @@
 <div class="card" xmlns="http://www.w3.org/1999/html">
-  <div class="card-header text-white question-header">
+  <div class="card-header text-white question-header cursor-pointer" (click)="model.isCollapsed = !model.isCollapsed">
```

Comment:
What's the difference using `triggerModelChange`? I don't think it matters for saving the model?

Reply:
If we don't fire event to change the model explicitly but change it silently like this. This will cause hard-to-find bug like this (more technically, the information flow between child and parent component is broken):

Do you expect the model to change if you do this?
```html
<tm-question-edit-form [formMode]="QuestionEditFormMode.EDIT"
                         [formModel]="questionEditFormModels[i]"></tm-question-edit-form>
```
Or this?
```html
<tm-question-edit-form [formMode]="QuestionEditFormMode.EDIT"
                         [(formModel)]="questionEditFormModels[i]"></tm-question-edit-form>
```
Or this?
```html
<tm-question-edit-form [formMode]="QuestionEditFormMode.EDIT"
                         [formModel]="questionEditFormModels[i]" (formModelChange)="questionEditFormModels[i] = $event"></tm-question-edit-form>
```
P.S. The behavior of the second and the third one are the same.

Reply:
So the point of emitting the event is to force Angular to detect the change and update the model right?

My thought was that since `saveExistingQuestionHandler` doesn't make use of the `isChanged` field, it wouldn't break anything, even if the parent and child are de-synced. But I guess it is better to be future-proof.